hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/DynamicConfigurationAwareConfig.java

```diff
@@ -174,6 +174,30 @@ public MapConfig getMapConfig(String name) {
         return getMapConfigInternal(name, name);
     }
 
+    @Override
+    public MapConfig getMapConfigOrNull(String name) {
+        return getMapConfigOrNullInternal(name);
+    }
+
+    private MapConfig getMapConfigOrNullInternal(String name) {
+        return getMapConfigOrNullInternal(name, name);
+    }
+
+    private MapConfig getMapConfigOrNullInternal(String name, String fallbackName) {
```

Comment:
Flavor warning. In case of such internal methods, I often add a 0 postfix. 

So instead of 

getMapConfigOrNullInternal

I call them

getMapConfigOrNull0

This is done frequently in the JDK.